earth_enterprise/src/SConstruct

```diff
@@ -424,6 +424,42 @@ def FusionVersionNumber(source):
     print 'Unable to find Fusion Version Number:', exp
 fusion_version_number = FusionVersionNumber('fusion_version.txt')
 
+def get_cc_version(cc_path):
```

Comment:
Weren't these already added in PR #476?  Are they now in this file twice?

Reply:
:P You're right. These were from different commits and they got automerged into repeated function definitions.